File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/d/ExecuteLoopDTO.java
package iih.ci.ord.dto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 执行闭环 DTO数据 
 * 
 */
public class ExecuteLoopDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 医嘱id
	 * @return String
	 */
	public String getId_or() {
		return ((String) getAttrVal("Id_or"));
	}
	/**
	 * 医嘱id
	 * @param Id_or
	 */
	public void setId_or(String Id_or) {
		setAttrVal("Id_or", Id_or);
	}
	/**
	 * 执行步骤
	 * @return FArrayList
	 */
	public FArrayList getLoops() {
		return ((FArrayList) getAttrVal("Loops"));
	}
	/**
	 * 执行步骤
	 * @param Loops
	 */
	public void setLoops(FArrayList Loops) {
		setAttrVal("Loops", Loops);
	}
	/**
	 * 服务项目
	 * @return FArrayList
	 */
	public FArrayList getSrvs() {
		return ((FArrayList) getAttrVal("Srvs"));
	}
	/**
	 * 服务项目
	 * @param Srvs
	 */
	public void setSrvs(FArrayList Srvs) {
		setAttrVal("Srvs", Srvs);
	}
	/**
	 * 计划执行时间
	 * @return FDateTime
	 */
	public FDateTime getDt_mp_plan() {
		return ((FDateTime) getAttrVal("Dt_mp_plan"));
	}
	/**
	 * 计划执行时间
	 * @param Dt_mp_plan
	 */
	public void setDt_mp_plan(FDateTime Dt_mp_plan) {
		setAttrVal("Dt_mp_plan", Dt_mp_plan);
	}
	/**
	 * 服务
	 * @return String
	 */
	public String getId_srv() {
		return ((String) getAttrVal("Id_srv"));
	}
	/**
	 * 服务
	 * @param Id_srv
	 */
	public void setId_srv(String Id_srv) {
		setAttrVal("Id_srv", Id_srv);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/d/ExecuteLoopDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.d
{
    /// <summary>
    /// ExecuteLoopDTO 的摘要说明。
    /// </summary>
    public class ExecuteLoopDTO : BaseDTO {

        public ExecuteLoopDTO() {
 
        }

        /// <summary>
        /// 医嘱id
        /// </summary>
		public string Id_or {
            get { return getAttrVal<string>("Id_or",null); }
            set { setAttrVal<string>("Id_or", value); }
        }

        /// <summary>
        /// 执行步骤
        /// </summary>
		public FArrayList Loops {
            get { return getAttrVal<FArrayList>("Loops",null); }
            set { setAttrVal<FArrayList>("Loops", value); }
        }

        /// <summary>
        /// 服务项目
        /// </summary>
		public FArrayList Srvs {
            get { return getAttrVal<FArrayList>("Srvs",null); }
            set { setAttrVal<FArrayList>("Srvs", value); }
        }

        /// <summary>
        /// 计划执行时间
        /// </summary>
		public DateTime? Dt_mp_plan {
            get { return getAttrVal<FDateTime>("Dt_mp_plan",null); }
            set { setAttrVal<FDateTime>("Dt_mp_plan", value); }
        }

        /// <summary>
        /// 服务
        /// </summary>
		public string Id_srv {
            get { return getAttrVal<string>("Id_srv",null); }
            set { setAttrVal<string>("Id_srv", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_or", "Loops", "Srvs", "Dt_mp_plan", "Id_srv"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.d.ExecuteLoopDTO";
        }
    }
}
